x-pack/plugins/remote_clusters/public/sections/remote_cluster_list/remote_cluster_table/remote_cluster_table.test.js

```diff
@@ -1,99 +0,0 @@
-/*
- * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
- * or more contributor license agreements. Licensed under the Elastic License;
- * you may not use this file except in compliance with the Elastic License.
- */
-
-import React from 'react';
-import { Provider } from 'react-redux';
-import { renderWithIntl, mountWithIntl } from 'test_utils/enzyme_helpers';
-import { findTestSubject } from '@elastic/eui/lib/test';
-
-import { remoteClustersStore } from '../../../store';
-import { RemoteClusterTable } from './remote_cluster_table';
-
-// Make sure we have deterministic aria IDs.
-jest.mock('@elastic/eui/lib/components/form/form_row/make_id', () => () => 'fakeId');
-
-jest.mock('../../../services', () => {
-  const services = require.requireActual('../../../services');
-  return {
-    ...services,
-    getRouterLinkProps: (link) => ({ href: link }),
-  };
-});
-
-describe('RemoteClusterTable', () => {
-  test('renders a row for an API-defined remote cluster', () => {
-    const clusters = [{
-      name: 'test-cluster',
-      seeds: ['seed', 'seed2'],
-    }];
-
-    const component = renderWithIntl(
-      <Provider store={remoteClustersStore}>
-        <RemoteClusterTable
-          clusters={clusters}
-          openDetailPanel={() => {}}
-        />
-      </Provider>
-    );
-
-    expect(component.find('tbody > tr').first()).toMatchSnapshot();
-  });
-
-  test('renders a row with a tooltip for a remote cluster defined in elasticsearch.yml', () => {
```

Comment:
I might have missed it, but is this assertion preserved anywhere?

Reply:
Good catch, I will add a test for it